docs/guis/multiturn/excitation.md

```diff
@@ -0,0 +1,207 @@
+    <!-- TODO: check this first! Most of these information can be found by searching `LSA` in the `CCM`, then navigating to and opening `Generator`, `Edit` mode and searching & selecting the relevant beam process. -->
+
+    An __example table__ is shown below.
+
+    | Time  | Energy (TeV) | Phase Knob | ATS  | Kick Amplitude (%) |   Optics    |
+    |:-----:|:------------:|:----------:|:----:|:------------------:|:-----------:|
+    |  30s  |     0.46     |    100%    |  1   |         3          | TODO: write |
+    | 240s  |     1.0      |    50%     |  1   |         7          | TODO: write |
+    | 405s  |     1.9      |     0%     |  1   |         13         | TODO: write |
+    | 580s  |     2.9      |     0%     |  1   |         19         | TODO: write |
+    | 720s  |     3.7      |     0%     |  1   |         24         | TODO: write |
+    | 860s  |     4.5      |     0%     |  1   |         30         | TODO: write |
+    | 1010s |     5.5      |     0%     | 0.75 |         36         | TODO: write |
+    | 1160s |     6.2      |     0%     | 0.57 |         41         | TODO: write |
+    | 1247s |     6.6      |     0%     | 0.5  |         45         | TODO: write |
+
+    The values in this table are a good starting point, but it is important to monitor the losses and reduce the kick amplitudes accordingly.
+
+??? info "Kick Amplitudes at Top Energy"
+
+    At top energy the increased beam rigidity allows us larger kick amplitudes.
+    A reasonable starting point is __5%__, then going up in steps of __5%__ until beam losses during kicks stop being reasonable.
+    Remember that it is very time-consuming to get back to this state when losing the beam at top energy, so monitor beam losses carefully and be reasonable with the kick increases.
+
+### Exciting the Beam
+
+Trigger an acquisition by clicking the yellow ++"Acquire with ACDipole excitation"++ button at the bottom left of the GUI.
+The AC Dipole will arm, then kick the beam.
+Make sure to have a `BLM Display` application open and to monitor the losses during that time.
+
+Afterwards, a new tab will open at the very top of the GUI to display the BPM measurements, which can be checked.
+
+??? info "Losses on Kicks"
+    Sometimes when increasing the kick amplitude, one will notice large losses.
+    In this case it is recommended to kick a couple times at this amplitude or just below to see if the losses reduce or are consistent.
+
+    Should they reduce the beam might have just needed cleaning and one can increase the kick amplitude further.
+    Otherwise, stop increasing unless a beam dump is affordable.
+    Refer to the experts on shift if unsure about the losses, and whether the kick amplitude can be increased further.
+
+!!! danger "Do not Kick Both Beams Simultaneously"
+    Triggering an acquisition will always turns off the tune feedback, radial loop, and orbit feedback for that beam.
```

Comment:
There are two things here:
a) do not kick both beams simultaneously, as this might exceed the losses threshold and dump the beams.
b) leave a few seconds between the kicks for the reasons mentioned here.